README.md

```diff
@@ -1,7 +1,35 @@
-# trustify
+# Trustify
 
 [![ci](https://github.com/trustification/trustify/actions/workflows/ci.yaml/badge.svg)](https://github.com/trustification/trustify/actions/workflows/ci.yaml)
 
+Welcome to Trustify, an open-source platform bringing together Software Bills of Materials (SBOMs) and security advisories to ensure software supply chain security threats and vulnerabilities are minimized and caught promptly.
```

Comment:
I prefer more assertive, succinct language ex.

"Trustify provides a single, searchable abstraction over all cyclonedx/spdx SBOMs - cross referencing against public security advisories to help identify and respond to software security threats & vulnerabilities."